src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/NativeLibrary.NativeAot.Windows.cs

```diff
@@ -35,6 +38,12 @@ private static IntPtr LoadLibraryHelper(string libraryName, int flags, ref LoadL
                 errorTracker.TrackErrorCode(Marshal.GetLastWin32Error());
```

Comment:
```suggestion
                errorTracker.TrackErrorCode(Marshal.GetLastPInvokeError());
```
Nit: We prefer GetLastPInvokeError over GetLastWin32Error. (This one was missed during the find&replace pass somehow.)